src/components/policy/policy_regular/src/policy_table/validation.cc

```diff
@@ -136,6 +136,24 @@ bool ModuleConfig::Validate() const {
       return false;
     }
   }
+
+  if (endpoints.empty()) {
```

Comment:
@AKalinich-Luxoft does this chek mandatory?
isn't template parameter typedef Array<String<10, 255>, 1, 3> URL; does not specify explicitly that length of this array should be greater than 0?